Dockerfile

```diff
@@ -20,24 +20,26 @@ RUN if [ "${DB_DRIVER}" = "mysql" ]; then apk add --no-cache mariadb mariadb-cli
 RUN if [ "${DB_DRIVER}" = "pgsql" ]; then apk add --no-cache postgresql postgresql-client php7-pdo_pgsql; fi
 RUN if [ "${DB_DRIVER}" = "sqlite" ]; then apk add --no-cache sqlite php7-pdo_sqlite; fi
 
+RUN ln -sf /dev/stdout /var/log/nginx/access.log && \
+    ln -sf /dev/stderr /var/log/nginx/error.log && \
+    ln -sf /dev/stdout /var/log/php7/error.log && \
+    ln -sf /dev/stderr /var/log/php7/error.log
+
+RUN rm -rf /var/www/html
+RUN mkdir -p /var/www/html
+
+
```

Comment:
What is the benefit of this change?

Reply:
As docker images can continue incrementally, I pulled up the commands that are likely not changing often/don't need to be rerun on config changes.

Reply:
I see. I didn't know that.